src/Identity/UI/src/Microsoft.AspNetCore.Identity.UI.csproj

```diff
@@ -7,23 +7,30 @@
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
     <PackageTags>aspnetcore;identity;membership;razorpages</PackageTags>
     <IsShippingPackage>true</IsShippingPackage>
-    <GenerateEmbeddedFilesManifest>true</GenerateEmbeddedFilesManifest>
     <ProvideApplicationPartFactoryAttributeTypeName>Microsoft.AspNetCore.Mvc.ApplicationParts.NullApplicationPartFactory, Microsoft.AspNetCore.Mvc.Core</ProvideApplicationPartFactoryAttributeTypeName>
     <RazorCompileOnBuild>false</RazorCompileOnBuild>
     <RazorCompileOnPublish>false</RazorCompileOnPublish>
     <EnableDefaultRazorGenerateItems>false</EnableDefaultRazorGenerateItems>
     <DisableFastUpToDateCheck>true</DisableFastUpToDateCheck>
     <AddRazorSupportForMvc>true</AddRazorSupportForMvc>
+
+    <DisableStaticWebAssetsBuildPropsFileGeneration>true</DisableStaticWebAssetsBuildPropsFileGeneration>
+    <StaticWebAssetsDisableProjectBuildPropsFileGeneration>true</StaticWebAssetsDisableProjectBuildPropsFileGeneration>
+
+    <GetCurrentProjectStaticWebAssetsDependsOn>
+      $(GetCurrentProjectStaticWebAssetsDependsOn);
+      _UpdatedIdentityUIStaticWebAssets
+    </GetCurrentProjectStaticWebAssetsDependsOn>
+
+    <IdentityUIFrameworkVersion Condition="'$(IdentityUIFrameworkVersion)' == ''">Bootstrap4</IdentityUIFrameworkVersion>
+
   </PropertyGroup>
 
   <ItemGroup>
-    <EmbeddedResource Include="wwwroot/**/*" />
-    <EmbeddedResource Remove="wwwroot/**/LICENSE*" />
-    <None Remove="wwwroot/**/LICENSE*" />
+    <Content Remove="@(Content)" />
+    <Content Include="wwwroot\**\*" Pack="true" PackagePath="staticwebassets" />
```

Comment:
Is this ("staticwebassets") going to appear in the path here (C:\Users*.nuget\packages\microsoft.aspnetcore.identity.ui\3.0.0-preview7.19320.1\**staticwebassets**\V4\Identity\lib\jquery-validation\dist\jquery.validate.min.js)? If so, maybe we should choose a shorter option `swa`? I understand we can't completely eliminate the `path-too-long problem`, but we can do our best to minimize the chance.

Reply:
We can't actually, due to the way this works during development. We add a static files provider pointing to `C:\Users*.nuget\packages\microsoft.aspnetcore.identity.ui\3.0.0-preview7.19320.1*staticwebassets`. We can only shorten the staticwebassets piece, but everything else must stay.

The issue with https://github.com/aspnet/AspNetCore/issues/11529 is that given there is no proper support for SWA today in this package, it ends up using the default configuration, which publishes them to `_content/microsoft.aspnetcore.identity.ui/V4\Identity\lib\jquery-validation\dist\jquery.validate.min.js`

With proper support for SWA, the publish path becomes 
`identity\lib\jquery-validation\dist\jquery.validate.min.js`

Which is way shorter. We could also get away with shortening the paths even more, as these only appear in the default layout. When the layout is overriden (most common case) it can simply use the assets in the template.

Reply:
That's good and my suggestion here was simply shortening that even further, nothing fancy